fiaas_deploy_daemon/deployer/kubernetes/deployment/deployer.py

```diff
@@ -146,6 +151,10 @@ def _make_volumes(self, app_spec):
 
     def _make_volume_mounts(self, app_spec):
         volume_mounts = []
+        if app_spec.config_maps:
```

Comment:
@mortenlj @gregjones I just committed this, but I dont think this works out of the box as Morten mentioned in the issue. I read it is possible to use `subPath` to create a file for the configMap but it will probably(?) break something.  Right now each env variable is mounted to its own file like this 
```
/run/config/fiaas $ ls
ENV_CONFIG1 ENV_CONFIG2 ENV_CONFIG3
```
If I understand it correctly with subPath each configMap would be its own file, maybe it would be possible to make it mount under `/var/run/fiaas/config/<app_name>`(default configMap) and `/var/run/fiaas/config/test1` if you specify a configMap in fiaas.yml. Again, probably some breaking changes..

Reply:
Another possiblity could be to leave the default configMap as is, and add the new ones under `/run/config/fiaas/<user_defined_configmap>`

Reply:
something like this:
```
if app_spec.config_maps:
            for config_map in app_spec.config_maps:
                volume_mounts.insert(0, VolumeMount(name="{}-config".format(config_map),
                                                    readOnly=True, mountPath="/var/run/config/fiaas/{}/".format(config_map)))
```

Reply:
See my comment further down.